documentation/faq.mdx

```diff
@@ -38,7 +38,7 @@ For several reasons, we are **not** accepting financial donations of any kind.
 
 ### Why was this project started?
 
-First and foremost, for fun and the challenge. We are primarily interested in learning how the original games were made, and understanding the technology behind them. The end result of a working PC port that can potentially support mod and such is very motivating and exciting -- but ultimately secondary.
+First and foremost, for fun and the challenge. We are primarily interested in learning how the original games from the trilogy were made, and understanding the technology behind them. The end result of a working PC port that can potentially support mod and such is very motivating and exciting -- but ultimately secondary.
```

Comment:
This blurb is unnecessary (everyone knows what "original games" refers to) and Jak X is not part of the trilogy.

Reply:
It may just be me, but I viewed "Jak X" as an original game :-) I'll remove it